lib/parser.js

```diff
@@ -28,20 +28,31 @@ const blockOption = seq(
 ).map(
   (val) => {
     return {
-      name: val[2], 
+      name: val[2],
       value: val[4],
     };
   }
 );
 const blockOptions = blockOption.many();
 const startOfHeading = startOfLine.then(lexeme(regex(/[\*]+/)));
+
+const noSpaceText = lexeme(regex(/[^:^\n]+/));
+const atom = lexeme(regex(/[a-z0-9%@#_]+/i));
+const tag = lexeme(regex(/[\s]*:/)).then(atom);
+const ttl = lexeme(regex(/[^:]*/));
+const tagForm = ttl.then(tag.many()).skip(colon.times(0,1));
+const tagList = lazy(() =>
+                     { return tagForm; });
+
 const heading = seq(
   startOfHeading, singleLineText
 ).map(
   (val) => {
+    let tag = tagList.parse(val[1]).value || [];
```

Comment:
@yorkie Do you means that write tags parser to an other separate file then we can just find tags without `parser.js`? Also, tags cannot be filed of `OutLineNode`?